themes/src/main/resources/theme/base/email/html/workflow-notification.ftl

```diff
@@ -2,22 +2,23 @@
 <@layout.emailLayout>
 <h2>${kcSanitize(msg(subjectKey, daysRemaining, reason))?no_esc}</h2>
 
-<p>Dear ${user.firstName!user.username},</p>
-
 <#if messageKey == "customMessage">
-<p>${kcSanitize(customMessage)?no_esc}</p>
+    <p>${kcSanitize(customMessage)?no_esc}</p>
 <#else>
-<p>${kcSanitize(msg(messageKey, daysRemaining, reason))?no_esc}</p>
-</#if>
+    <p>Dear ${user.firstName!user.username},</p>
 
-<#if daysRemaining gt 0>
-<p><strong>Time remaining: ${daysRemaining} day<#if daysRemaining != 1>s</#if></strong></p>
-</#if>
+    <p>${kcSanitize(msg(messageKey, daysRemaining, reason))?no_esc}</p>
 
-<p>If you have questions, please contact your ${realmName} administrator.</p>
+    <#if daysRemaining gt 0>
+        <p><strong>Time remaining: ${daysRemaining} day<#if daysRemaining != 1>s</#if></strong></p>
+    </#if>
+
+    <p>If you have questions, please contact your ${realmName} administrator.</p>
+
+    <p>
+        Best regards,<br>
+        ${realmName} Administration
+    </p>
+</#if>
```

Comment:
I recall Stian suggesting that the e-mail should be more direct and make it clear that it is coming from an automated workflow. So no "Dear subject, "  or "Best regards," in the message. I don't recall exactly where we discussed this though.